docs/saml.md

```diff
@@ -0,0 +1,29 @@
+# SAML SSO configuration
+
+SAML SSO is available for users of us.prairielearn.com and ca.prairielearn.com. Reach out to support@prairielearn.com to get it set up.
+
+## Required attributes
+
+PrairieLearn requires that SAML identity providers (IdPs) make three attributes available.
+
+### Name
+
+The full name of the user, e.g. "Joe Smith". This attribute is often named `displayName` or `urn:oid:2.16.840.1.113730.3.1.241`.
```

Comment:
Is it worth commenting that the choice of format (first last vs last, first) and the use of formal vs preferred name is at the discretion of the institution? Maybe comment that this name is shown in gradebooks so recommend that it matches the format instructors would usually see in other systems e.g., LMS.